cfgov/unprocessed/css/misc.less

```diff
@@ -655,6 +655,77 @@
     }
 }
 
+
+/* topdoc
+  name: Transition utility classes.
+  family: cfgov-misc
+  notes:
+    - "Adds transitions utilty classes for transform, opacity,
+       and for the removing the transition duration."
+  tags:
+    - cfgov-misc
+*/
+.u-move-transition {
+    transition: transform 0.25s ease-out;
+}
+
+.u-alpha-transition {
+    transition: opacity 0.25s ease-out;
+}
+
+.u-no-animation {
+    transition-duration: 0s;
+}
+
+/* topdoc
+  name: Move utility classes.
+  family: cfgov-misc
+  notes:
+    - "Utility classes for moving an element using transform translate values."
+  tags:
+    - cfgov-misc
+*/
+.u-move-to-origin {
+    transform: translateX( 0 );
+}
+
+.u-move-left {
+    transform: translateX( -100% );
+}
+
+.u-move-left-2x {
+    transform: translateX( -200% );
```

Comment:
Yeah without editing the CSS from JS, I couldn't see a way around this (and I think it would be preferable to keep the JS/CSS separate). Let's re-visit when the full menu is in and maybe you'll see another way around it.